next_frontend/components/common/NewLinkForm.tsx

```diff
@@ -79,6 +77,8 @@ const NewLinkForm: React.FC<NewLinkFormProps> = ({
 
     result.data.hidden = findSiblingHiddenState(assuranceCase, node.data.id)
 
+    console.log('RESULT', result)
```

Comment:
Can you please remove these logs? Console is really crowded when debugging issues